src/main/assembly/dist-assembly.xml

```diff
@@ -198,6 +198,8 @@
                 <include>org.apache.logging.log4j:*:jar</include>
                 <include>org.slf4j:*:jar</include>
                 <include>com.github.javafaker:*:jar</include>
+                <include>net.datafaker:*:jar</include>
```

Comment:
Great you took care on this ... please also add these dependencies to our Enterprise Edition